__tests__/dataGenerator.test.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose');
+const app = require('../server/index.js');
```

Comment:
After taking a look at the seed code: The tests here are written specifically to check if seeding was successful for a Mongo database. While you _could_ come back here and edit the tests every time you change databases, maybe it would be worth considering to test the `seed` function from around line 202 in `seed.js` directly, then separately testing the database `insertMany` functions below it when you implement them later. It's another case for more modularity :)